spec/listformat.html

```diff
@@ -0,0 +1,395 @@
+          1. Let _first_ be a new Record { [[Type]]: *"element"*, [[Value]]: _list_[0] }.
+          1. Let _second_ be a new Record { [[Type]]: *"element"*, [[Value]]: _list_[1] }.
+          1. Let _placeables_ be a new Record { [[0]]: _first_, [[1]]: _second_ }.
+          1. Return DeconstructPattern(_pattern_, _placeables_).
+        1. Let _last_ be a new Record { [[Type]]: *"element"*, [[Value]]: _list_[_size_ - 1] }.
+        1. Let _parts_ be &laquo; _last_ &raquo;.
+        1. Let _i_ be _size_ - 2.
+        1. Repeat, while _i_ &ge; 0
+          1. Let _n_ be an implementation-defined index into _listFormat_.[[Templates]] based on _listFormat_.[[Locale]], _head_, and _parts_.
+          1. If _i_ is 0, then
+            1. Let _pattern_ be _listFormat_.[[Templates]][_n_].[[Start]].
+          1. Else, if _i_ is less than _size_ - 2, then
+            1. Let _pattern_ be _listFormat_.[[Templates]][_n_].[[Middle]].
+          1. Else,
+            1. Let _pattern_ be _listFormat_.[[Templates]][_n_].[[End]].
+          1. Let _head_ be a new Record { [[Type]]: *"element"*, [[Value]]: _list_[_i_] }.
+          1. Let _placeables_ be a new Record { [[0]]: _head_, [[1]]: _parts_ }.
+          1. Set _parts_ to DeconstructPattern(_pattern_, _placeables_).
+          1. Decrement _i_ by 1.
+        1. Return _parts_.
+      </emu-alg>
+
+      <emu-note>
+        The index _n_ to select across multiple templates permits the conjunction to be dependent on the context, as in Spanish, where "y" or "e" may be selected, dependending on the following word.
+      </emu-note>
+    </emu-clause>
+
+    <emu-clause id="sec-formatlist" aoid="FormatList">
+      <h1>FormatList ( _listFormat_, _list_ )</h1>
+
+      <p>
+        The FormatList abstract operation is called with arguments _listFormat_ (which must be an object initialized as a ListFormat) and _list_ (which must be a List of String values), and performs the following steps:
+      </p>
+
+      <emu-alg>
+        1. Let _parts_ be CreatePartsFromList(_listFormat_, _list_).
+        1. Let _result_ be an empty String.
+        1. For each _part_ in _parts_, do
+          1. Set _result_ to a String value produced by concatenating _result_ and _part_.[[Value]].
+        1. Return _result_.
+      </emu-alg>
+
+    </emu-clause>
+
+    <emu-clause id="sec-formatlisttoparts" aoid="FormatListToParts">
+      <h1>FormatListToParts ( _listFormat_, _list_ )</h1>
+
+      <p>
+        The FormatListToParts abstract operation is called with arguments _listFormat_ (which must be an object initialized as a ListFormat) and _list_ (which must be a List of String values), and performs the following steps:
+      </p>
+
+      <emu-alg>
+        1. Let _parts_ be CreatePartsFromList(_listFormat_, _list_).
+        1. Let _result_ be ArrayCreate(0).
+        1. Let _n_ be 0.
+        1. For each _part_ in _parts_, do
+          1. Let _O_ be ObjectCreate(%ObjectPrototype%).
```

Comment:
```suggestion
          1. Let _O_ be ObjectCreate(%Object.prototype%).
```

Also, we need to update ObjectCreate in the whole spec text, as a follow up. It's now used as `OrdinaryObjectCreate`.